src/deploydocs.jl

```diff
@@ -141,6 +142,8 @@ It defaults to the value of `branch`.
 **`repo_previews`** is the remote repository to which pull request previews are
 deployed. It defaults to the value of `repo`.
 
+** `deploy_repo`** is the remote repository to deploy to.
```

Comment:
The extra space between `**` and the backtick mess up the formatting of how this looks in the rendered documentation. It should come before the two notes, which relate to previews.

Also, I would expand the docstring a bit to make it clear what the expected format is and the default value. Presumably, "The format is the same as that of `repo`. Defaults to the value of `repo`. Actually, the format requirements should probably also be specified for the `repo_previews` that precedes it. Might as well fix that in this PR.

Reply:
Other than that, LGTM